components/form/fields/RatingField.tsx

```diff
@@ -0,0 +1,23 @@
+import { Rating as MuiRatingField, Typography } from '@mui/material'
+
+type RatingFieldProps = {
+    name: string
+    field: any
+    onChange: any
+}
+
+const RatingField = ({ name, field, onChange }: RatingFieldProps) => {
+    const parseValue = parseInt(field?.value)
```

Comment:
No need to parseInt, we will be ensuring this is a `number` type

Reply:
The way the mui rating field return the value is as a string. In order to get it as a number type it would  need its own onChange handler which uses the newValue.

`const handleChange = (event, newValue) => {
return newValue;
}`

but i think that instead of adding this into the useForm hook and making the check every time we use it on all fields . we just slide in a nice little parseInt which will only be used when someone clicks a start and not all the time.